spec/models/location_spec.rb

```diff
@@ -6,4 +6,36 @@
   it { should validate_presence_of(:address) }
   it { should validate_presence_of(:user) }
   it { should validate_presence_of(:zipcode) }
+
+  describe "#supported?" do
+    context "when their zipcode is supported" do
+      it "returns true" do
+        location = build(:location, :supported)
+
+        supported = location.supported?
+
+        expect(supported).to be true
```

Comment:
I tend to favor `.to be true` so that we can maintain the three-step setup-exercise-assert test format